[BUG] "value" field in init expression of @Getter(lazy=true) not working

This code:
```
@RequiredArgsConstructor
public class LazyGetterBugExample {

  String value;

  @Getter(lazy = true)
  int valueLength = value.length();

  public static void main(String[] args) {
    System.out.println(
        new LazyGetterBugExample("Lorem ipsum dolor sit amet")
            .getValueLength());
  }
}
```
will rise exception through compile time:

> error: cannot find symbol
  `@Getter(lazy = true)`
  ^
  symbol:   method length()
  location: variable value of type java.lang.Object

..., but if we just RENAME field variable "value" to whatever else - "val", for example - it compiles and executes successfully!

**The reason is...**
this class will delomboked to this:
```
public int getValueLength() {
    java.lang.Object value = this.valueLength.get();
    if (value == null) {
      //...
      final int actualValue = val.length();
      //...
    }
  }
```
I.e., when we try to use "value", it conflicts with local variable - "value", used in code, generated by Lombok (and has a type java.lang.Object and actually has not "length()" method, that's why we had an _error message_ above).

So, I think, the problem is that **_LOCAL VARIABLE_** in method, generated by Lombok for `@Getter(lazy = true)`, was not named so well...
This name - "value" - is used very often for field, so I think this bug is important to fix.

**Possible solution**
I think, there is 2 variants:
1. Insert check in code of generator, that if we use the "value" field in expression, then assign the local variable any another  name (and check that it is not used in expression too. of course)
2. Just rename the "value" local variable to something, that will never used by anyone - like `$_$321_IT_WILL_NEVER_USED_BY_ANYONE_123_$_$`, for example...

**Version info:**
 - Lombok version: 1.18.20
 - Platform: javac, full `$ java -version` output:
openjdk version "16.0.2" 2021-07-20
OpenJDK Runtime Environment Corretto-16.0.2.7.1 (build 16.0.2+7)
OpenJDK 64-Bit Server VM Corretto-16.0.2.7.1 (build 16.0.2+7, mixed mode, sharing)

--- a/src/core/lombok/eclipse/handlers/HandleGetter.java
+++ b/src/core/lombok/eclipse/handlers/HandleGetter.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2009-2022 The Project Lombok Authors.
+ * Copyright (C) 2009-2024 The Project Lombok Authors.
  * 
  * Permission is hereby granted, free of charge, to any person obtaining a copy
  * of this software and associated documentation files (the "Software"), to deal
@@ -323,7 +323,7 @@ public class HandleGetter extends EclipseAnnotationHandler<Getter> {
 		TYPE_MAP = Collections.unmodifiableMap(m);
 	}
 	
-	private static char[] valueName = "value".toCharArray();
+	private static char[] valueName = "$value".toCharArray();
 	private static char[] actualValueName = "actualValue".toCharArray();
 	
 	private static final int PARENTHESIZED = (1 << ASTNode.ParenthesizedSHIFT) & ASTNode.ParenthesizedMASK;

--- a/src/core/lombok/javac/handlers/HandleGetter.java
+++ b/src/core/lombok/javac/handlers/HandleGetter.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2009-2022 The Project Lombok Authors.
+ * Copyright (C) 2009-2024 The Project Lombok Authors.
  * 
  * Permission is hereby granted, free of charge, to any person obtaining a copy
  * of this software and associated documentation files (the "Software"), to deal
@@ -358,7 +358,7 @@ public class HandleGetter extends JavacAnnotationHandler<Getter> {
 		}
 		if (copyOfBoxedFieldType == null) copyOfBoxedFieldType = copyType(maker, field, source);
 		
-		Name valueName = fieldNode.toName("value");
+		Name valueName = fieldNode.toName("$value");
 		Name actualValueName = fieldNode.toName("actualValue");
 		
 		/* java.lang.Object value = this.fieldName.get();*/ {

--- a/test/transform/resource/after-delombok/DelegateOnGetter.java
+++ b/test/transform/resource/after-delombok/DelegateOnGetter.java
@@ -6,11 +6,11 @@ class DelegateOnGetter {
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public Bar getBar() {
-		java.lang.Object value = this.bar.get();
-		if (value == null) {
+		java.lang.Object $value = this.bar.get();
+		if ($value == null) {
 			synchronized (this.bar) {
-				value = this.bar.get();
-				if (value == null) {
+				$value = this.bar.get();
+				if ($value == null) {
 					final Bar actualValue = new Bar() {
 						public void setList(java.util.ArrayList<String> list) {
 						}
@@ -18,12 +18,12 @@ class DelegateOnGetter {
 							return 42;
 						}
 					};
-					value = actualValue == null ? this.bar : actualValue;
-					this.bar.set(value);
+					$value = actualValue == null ? this.bar : actualValue;
+					this.bar.set($value);
 				}
 			}
 		}
-		return (Bar) (value == this.bar ? null : value);
+		return (Bar) ($value == this.bar ? null : $value);
 	}
 	@java.lang.SuppressWarnings("all")
 	public void setList(final java.util.ArrayList<java.lang.String> list) {

--- a/test/transform/resource/after-delombok/GetterLazy.java
+++ b/test/transform/resource/after-delombok/GetterLazy.java
@@ -4,17 +4,17 @@ class GetterLazy {
 	private final java.util.concurrent.atomic.AtomicReference<java.lang.Object> fieldName = new java.util.concurrent.atomic.AtomicReference<java.lang.Object>();
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public ValueType getFieldName() {
-		java.lang.Object value = this.fieldName.get();
-		if (value == null) {
+		java.lang.Object $value = this.fieldName.get();
+		if ($value == null) {
 			synchronized (this.fieldName) {
-				value = this.fieldName.get();
-				if (value == null) {
+				$value = this.fieldName.get();
+				if ($value == null) {
 					final ValueType actualValue = new ValueType();
-					value = actualValue == null ? this.fieldName : actualValue;
-					this.fieldName.set(value);
+					$value = actualValue == null ? this.fieldName : actualValue;
+					this.fieldName.set($value);
 				}
 			}
 		}
-		return (ValueType) (value == this.fieldName ? null : value);
+		return (ValueType) ($value == this.fieldName ? null : $value);
 	}
 }

--- a/test/transform/resource/after-delombok/GetterLazyArguments.java
+++ b/test/transform/resource/after-delombok/GetterLazyArguments.java
@@ -21,99 +21,99 @@ class GetterLazyArguments {
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField1() {
-		java.lang.Object value = this.field1.get();
-		if (value == null) {
+		java.lang.Object $value = this.field1.get();
+		if ($value == null) {
 			synchronized (this.field1) {
-				value = this.field1.get();
-				if (value == null) {
+				$value = this.field1.get();
+				if ($value == null) {
 					final String actualValue = stringInt(("a"), (1));
-					value = actualValue == null ? this.field1 : actualValue;
-					this.field1.set(value);
+					$value = actualValue == null ? this.field1 : actualValue;
+					this.field1.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field1 ? null : value);
+		return (String) ($value == this.field1 ? null : $value);
 	}
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField2() {
-		java.lang.Object value = this.field2.get();
-		if (value == null) {
+		java.lang.Object $value = this.field2.get();
+		if ($value == null) {
 			synchronized (this.field2) {
-				value = this.field2.get();
-				if (value == null) {
+				$value = this.field2.get();
+				if ($value == null) {
 					final String actualValue = stringInt(true ? "a" : "b", true ? 1 : 0);
-					value = actualValue == null ? this.field2 : actualValue;
-					this.field2.set(value);
+					$value = actualValue == null ? this.field2 : actualValue;
+					this.field2.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field2 ? null : value);
+		return (String) ($value == this.field2 ? null : $value);
 	}
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField3() {
-		java.lang.Object value = this.field3.get();
-		if (value == null) {
+		java.lang.Object $value = this.field3.get();
+		if ($value == null) {
 			synchronized (this.field3) {
-				value = this.field3.get();
-				if (value == null) {
+				$value = this.field3.get();
+				if ($value == null) {
 					final String actualValue = stringInt(("a"), true ? 1 : 0);
-					value = actualValue == null ? this.field3 : actualValue;
-					this.field3.set(value);
+					$value = actualValue == null ? this.field3 : actualValue;
+					this.field3.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field3 ? null : value);
+		return (String) ($value == this.field3 ? null : $value);
 	}
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField4() {
-		java.lang.Object value = this.field4.get();
-		if (value == null) {
+		java.lang.Object $value = this.field4.get();
+		if ($value == null) {
 			synchronized (this.field4) {
-				value = this.field4.get();
-				if (value == null) {
+				$value = this.field4.get();
+				if ($value == null) {
 					final String actualValue = stringRunnable(fun(), () -> {
 					});
-					value = actualValue == null ? this.field4 : actualValue;
-					this.field4.set(value);
+					$value = actualValue == null ? this.field4 : actualValue;
+					this.field4.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field4 ? null : value);
+		return (String) ($value == this.field4 ? null : $value);
 	}
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField5() {
-		java.lang.Object value = this.field5.get();
-		if (value == null) {
+		java.lang.Object $value = this.field5.get();
+		if ($value == null) {
 			synchronized (this.field5) {
-				value = this.field5.get();
-				if (value == null) {
+				$value = this.field5.get();
+				if ($value == null) {
 					final String actualValue = stringRunnable(("a"), () -> {
 					});
-					value = actualValue == null ? this.field5 : actualValue;
-					this.field5.set(value);
+					$value = actualValue == null ? this.field5 : actualValue;
+					this.field5.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field5 ? null : value);
+		return (String) ($value == this.field5 ? null : $value);
 	}
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField6() {
-		java.lang.Object value = this.field6.get();
-		if (value == null) {
+		java.lang.Object $value = this.field6.get();
+		if ($value == null) {
 			synchronized (this.field6) {
-				value = this.field6.get();
-				if (value == null) {
+				$value = this.field6.get();
+				if ($value == null) {
 					final String actualValue = true ? stringInt(true ? "a" : "b", true ? 1 : 0) : "";
-					value = actualValue == null ? this.field6 : actualValue;
-					this.field6.set(value);
+					$value = actualValue == null ? this.field6 : actualValue;
+					this.field6.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field6 ? null : value);
+		return (String) ($value == this.field6 ? null : $value);
 	}
 }

--- a/test/transform/resource/after-delombok/GetterLazyBoolean.java
+++ b/test/transform/resource/after-delombok/GetterLazyBoolean.java
@@ -33,32 +33,32 @@ class GetterLazyBoolean {
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public boolean isBooleanValue() {
-		java.lang.Object value = this.booleanValue.get();
-		if (value == null) {
+		java.lang.Object $value = this.booleanValue.get();
+		if ($value == null) {
 			synchronized (this.booleanValue) {
-				value = this.booleanValue.get();
-				if (value == null) {
+				$value = this.booleanValue.get();
+				if ($value == null) {
 					final boolean actualValue = calculateBoolean();
-					value = actualValue;
-					this.booleanValue.set(value);
+					$value = actualValue;
+					this.booleanValue.set($value);
 				}
 			}
 		}
-		return (java.lang.Boolean) value;
+		return (java.lang.Boolean) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public boolean isOtherBooleanValue() {
-		java.lang.Object value = this.otherBooleanValue.get();
-		if (value == null) {
+		java.lang.Object $value = this.otherBooleanValue.get();
+		if ($value == null) {
 			synchronized (this.otherBooleanValue) {
-				value = this.otherBooleanValue.get();
-				if (value == null) {
+				$value = this.otherBooleanValue.get();
+				if ($value == null) {
 					final boolean actualValue = !calculateBoolean();
-					value = actualValue;
-					this.otherBooleanValue.set(value);
+					$value = actualValue;
+					this.otherBooleanValue.set($value);
 				}
 			}
 		}
-		return (java.lang.Boolean) value;
+		return (java.lang.Boolean) $value;
 	}
 }

--- a/test/transform/resource/after-delombok/GetterLazyEahcToString.java
+++ b/test/transform/resource/after-delombok/GetterLazyEahcToString.java
@@ -46,18 +46,18 @@ class GetterLazyEahcToString {
 	
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getValue() {
-		java.lang.Object value = this.value.get();
-		if (value == null) {
+		java.lang.Object $value = this.value.get();
+		if ($value == null) {
 			synchronized (this.value) {
-				value = this.value.get();
-				if (value == null) {
+				$value = this.value.get();
+				if ($value == null) {
 					final String actualValue = "";
-					value = actualValue == null ? this.value : actualValue;
-					this.value.set(value);
+					$value = actualValue == null ? this.value : actualValue;
+					this.value.set($value);
 				}
 			}
 		}
-		return (String) (value == this.value ? null : value);
+		return (String) ($value == this.value ? null : $value);
 	}
 	
 	@java.lang.SuppressWarnings("all")

--- a/test/transform/resource/after-delombok/GetterLazyErrorPosition.java
+++ b/test/transform/resource/after-delombok/GetterLazyErrorPosition.java
@@ -3,17 +3,17 @@ class GetterLazyErrorPosition {
 
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public String getField() {
-		java.lang.Object value = this.field.get();
-		if (value == null) {
+		java.lang.Object $value = this.field.get();
+		if ($value == null) {
 			synchronized (this.field) {
-				value = this.field.get();
-				if (value == null) {
+				$value = this.field.get();
+				if ($value == null) {
 					final String actualValue = true ? "" : new ErrorPosition();
-					value = actualValue == null ? this.field : actualValue;
-					this.field.set(value);
+					$value = actualValue == null ? this.field : actualValue;
+					this.field.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field ? null : value);
+		return (String) ($value == this.field ? null : $value);
 	}
 }

--- a/test/transform/resource/after-delombok/GetterLazyGenerics.java
+++ b/test/transform/resource/after-delombok/GetterLazyGenerics.java
@@ -6,32 +6,32 @@ class GetterLazyGenerics<E> {
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public E getField() {
-		java.lang.Object value = this.field.get();
-		if (value == null) {
+		java.lang.Object $value = this.field.get();
+		if ($value == null) {
 			synchronized (this.field) {
-				value = this.field.get();
-				if (value == null) {
+				$value = this.field.get();
+				if ($value == null) {
 					final E actualValue = getAny();
-					value = actualValue == null ? this.field : actualValue;
-					this.field.set(value);
+					$value = actualValue == null ? this.field : actualValue;
+					this.field.set($value);
 				}
 			}
 		}
-		return (E) (value == this.field ? null : value);
+		return (E) ($value == this.field ? null : $value);
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public long getField2() {
-		java.lang.Object value = this.field2.get();
-		if (value == null) {
+		java.lang.Object $value = this.field2.get();
+		if ($value == null) {
 			synchronized (this.field2) {
-				value = this.field2.get();
-				if (value == null) {
+				$value = this.field2.get();
+				if ($value == null) {
 					final long actualValue = System.currentTimeMillis();
-					value = actualValue;
-					this.field2.set(value);
+					$value = actualValue;
+					this.field2.set($value);
 				}
 			}
 		}
-		return (java.lang.Long) value;
+		return (java.lang.Long) $value;
 	}
 }

--- a/test/transform/resource/after-delombok/GetterLazyInAnonymousClass.java
+++ b/test/transform/resource/after-delombok/GetterLazyInAnonymousClass.java
@@ -6,18 +6,18 @@ public class GetterLazyInAnonymousClass {
 
 			@java.lang.SuppressWarnings({"all", "unchecked"})
 			public String getString() {
-				java.lang.Object value = this.string.get();
-				if (value == null) {
+				java.lang.Object $value = this.string.get();
+				if ($value == null) {
 					synchronized (this.string) {
-						value = this.string.get();
-						if (value == null) {
+						$value = this.string.get();
+						if ($value == null) {
 							final String actualValue = "test";
-							value = actualValue == null ? this.string : actualValue;
-							this.string.set(value);
+							$value = actualValue == null ? this.string : actualValue;
+							this.string.set($value);
 						}
 					}
 				}
-				return (String) (value == this.string ? null : value);
+				return (String) ($value == this.string ? null : $value);
 			}
 		}
 	};

--- a/test/transform/resource/after-delombok/GetterLazyNative.java
+++ b/test/transform/resource/after-delombok/GetterLazyNative.java
@@ -10,137 +10,137 @@ class GetterLazyNative {
 	private final java.util.concurrent.atomic.AtomicReference<java.lang.Object> intArrayField = new java.util.concurrent.atomic.AtomicReference<java.lang.Object>();
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public boolean isBooleanField() {
-		java.lang.Object value = this.booleanField.get();
-		if (value == null) {
+		java.lang.Object $value = this.booleanField.get();
+		if ($value == null) {
 			synchronized (this.booleanField) {
-				value = this.booleanField.get();
-				if (value == null) {
+				$value = this.booleanField.get();
+				if ($value == null) {
 					final boolean actualValue = true;
-					value = actualValue;
-					this.booleanField.set(value);
+					$value = actualValue;
+					this.booleanField.set($value);
 				}
 			}
 		}
-		return (java.lang.Boolean) value;
+		return (java.lang.Boolean) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public byte getByteField() {
-		java.lang.Object value = this.byteField.get();
-		if (value == null) {
+		java.lang.Object $value = this.byteField.get();
+		if ($value == null) {
 			synchronized (this.byteField) {
-				value = this.byteField.get();
-				if (value == null) {
+				$value = this.byteField.get();
+				if ($value == null) {
 					final byte actualValue = 1;
-					value = actualValue;
-					this.byteField.set(value);
+					$value = actualValue;
+					this.byteField.set($value);
 				}
 			}
 		}
-		return (java.lang.Byte) value;
+		return (java.lang.Byte) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public short getShortField() {
-		java.lang.Object value = this.shortField.get();
-		if (value == null) {
+		java.lang.Object $value = this.shortField.get();
+		if ($value == null) {
 			synchronized (this.shortField) {
-				value = this.shortField.get();
-				if (value == null) {
+				$value = this.shortField.get();
+				if ($value == null) {
 					final short actualValue = 1;
-					value = actualValue;
-					this.shortField.set(value);
+					$value = actualValue;
+					this.shortField.set($value);
 				}
 			}
 		}
-		return (java.lang.Short) value;
+		return (java.lang.Short) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public int getIntField() {
-		java.lang.Object value = this.intField.get();
-		if (value == null) {
+		java.lang.Object $value = this.intField.get();
+		if ($value == null) {
 			synchronized (this.intField) {
-				value = this.intField.get();
-				if (value == null) {
+				$value = this.intField.get();
+				if ($value == null) {
 					final int actualValue = 1;
-					value = actualValue;
-					this.intField.set(value);
+					$value = actualValue;
+					this.intField.set($value);
 				}
 			}
 		}
-		return (java.lang.Integer) value;
+		return (java.lang.Integer) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public long getLongField() {
-		java.lang.Object value = this.longField.get();
-		if (value == null) {
+		java.lang.Object $value = this.longField.get();
+		if ($value == null) {
 			synchronized (this.longField) {
-				value = this.longField.get();
-				if (value == null) {
+				$value = this.longField.get();
+				if ($value == null) {
 					final long actualValue = 1;
-					value = actualValue;
-					this.longField.set(value);
+					$value = actualValue;
+					this.longField.set($value);
 				}
 			}
 		}
-		return (java.lang.Long) value;
+		return (java.lang.Long) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public float getFloatField() {
-		java.lang.Object value = this.floatField.get();
-		if (value == null) {
+		java.lang.Object $value = this.floatField.get();
+		if ($value == null) {
 			synchronized (this.floatField) {
-				value = this.floatField.get();
-				if (value == null) {
+				$value = this.floatField.get();
+				if ($value == null) {
 					final float actualValue = 1.0F;
-					value = actualValue;
-					this.floatField.set(value);
+					$value = actualValue;
+					this.floatField.set($value);
 				}
 			}
 		}
-		return (java.lang.Float) value;
+		return (java.lang.Float) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public double getDoubleField() {
-		java.lang.Object value = this.doubleField.get();
-		if (value == null) {
+		java.lang.Object $value = this.doubleField.get();
+		if ($value == null) {
 			synchronized (this.doubleField) {
-				value = this.doubleField.get();
-				if (value == null) {
+				$value = this.doubleField.get();
+				if ($value == null) {
 					final double actualValue = 1.0;
-					value = actualValue;
-					this.doubleField.set(value);
+					$value = actualValue;
+					this.doubleField.set($value);
 				}
 			}
 		}
-		return (java.lang.Double) value;
+		return (java.lang.Double) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public char getCharField() {
-		java.lang.Object value = this.charField.get();
-		if (value == null) {
+		java.lang.Object $value = this.charField.get();
+		if ($value == null) {
 			synchronized (this.charField) {
-				value = this.charField.get();
-				if (value == null) {
+				$value = this.charField.get();
+				if ($value == null) {
 					final char actualValue = '1';
-					value = actualValue;
-					this.charField.set(value);
+					$value = actualValue;
+					this.charField.set($value);
 				}
 			}
 		}
-		return (java.lang.Character) value;
+		return (java.lang.Character) $value;
 	}
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public int[] getIntArrayField() {
-		java.lang.Object value = this.intArrayField.get();
-		if (value == null) {
+		java.lang.Object $value = this.intArrayField.get();
+		if ($value == null) {
 			synchronized (this.intArrayField) {
-				value = this.intArrayField.get();
-				if (value == null) {
+				$value = this.intArrayField.get();
+				if ($value == null) {
 					final int[] actualValue = new int[] {1};
-					value = actualValue == null ? this.intArrayField : actualValue;
-					this.intArrayField.set(value);
+					$value = actualValue == null ? this.intArrayField : actualValue;
+					this.intArrayField.set($value);
 				}
 			}
 		}
-		return (int[]) (value == this.intArrayField ? null : value);
+		return (int[]) ($value == this.intArrayField ? null : $value);
 	}
 }

--- a/test/transform/resource/after-delombok/GetterLazyTransient.java
+++ b/test/transform/resource/after-delombok/GetterLazyTransient.java
@@ -4,18 +4,18 @@ class GetterLazyTransient {
 	private final transient int nonLazyTransientField = 3;
 	@java.lang.SuppressWarnings({"all", "unchecked"})
 	public int getNonTransientField() {
-		java.lang.Object value = this.nonTransientField.get();
-		if (value == null) {
+		java.lang.Object $value = this.nonTransientField.get();
+		if ($value == null) {
 			synchronized (this.nonTransientField) {
-				value = this.nonTransientField.get();
-				if (value == null) {
+				$value = this.nonTransientField.get();
+				if ($value == null) {
 					final int actualValue = 1;
-					value = actualValue;
-					this.nonTransientField.set(value);
+					$value = actualValue;
+					this.nonTransientField.set($value);
 				}
 			}
 		}
-		return (java.lang.Integer) value;
+		return (java.lang.Integer) $value;
 	}
 	@java.lang.SuppressWarnings("all")
 	public int getNonLazyTransientField() {

--- a/test/transform/resource/after-delombok/SkipSuppressWarnings.java
+++ b/test/transform/resource/after-delombok/SkipSuppressWarnings.java
@@ -8,17 +8,17 @@ class SkipSuppressWarnings {
 
 	@java.lang.SuppressWarnings({"unchecked"})
 	public String getField2() {
-		java.lang.Object value = this.field2.get();
-		if (value == null) {
+		java.lang.Object $value = this.field2.get();
+		if ($value == null) {
 			synchronized (this.field2) {
-				value = this.field2.get();
-				if (value == null) {
+				$value = this.field2.get();
+				if ($value == null) {
 					final String actualValue = "";
-					value = actualValue == null ? this.field2 : actualValue;
-					this.field2.set(value);
+					$value = actualValue == null ? this.field2 : actualValue;
+					this.field2.set($value);
 				}
 			}
 		}
-		return (String) (value == this.field2 ? null : value);
+		return (String) ($value == this.field2 ? null : $value);
 	}
 }

--- a/test/transform/resource/after-ecj/DelegateOnGetter.java
+++ b/test/transform/resource/after-ecj/DelegateOnGetter.java
@@ -10,13 +10,13 @@ class DelegateOnGetter {
     super();
   }
   public @Delegate @java.lang.SuppressWarnings({"all", "unchecked"}) Bar getBar() {
-    java.lang.Object value = this.bar.get();
-    if ((value == null))
+    java.lang.Object $value = this.bar.get();
+    if (($value == null))
         {
           synchronized (this.bar)
             {
-              value = this.bar.get();
-              if ((value == null))
+              $value = this.bar.get();
+              if (($value == null))
                   {
                     final Bar actualValue = new Bar() {
                       x() {
@@ -28,12 +28,12 @@ class DelegateOnGetter {
                         return 42;
                       }
                     };
-                    value = ((actualValue == null) ? this.bar : actualValue);
-                    this.bar.set(value);
+                    $value = ((actualValue == null) ? this.bar : actualValue);
+                    this.bar.set($value);
                   }
             }
         }
-    return (Bar) ((value == this.bar) ? null : value);
+    return (Bar) (($value == this.bar) ? null : $value);
   }
   public @java.lang.SuppressWarnings("all") int getInt() {
     return this.getBar().getInt();

--- a/test/transform/resource/after-ecj/GetterLazy.java
+++ b/test/transform/resource/after-ecj/GetterLazy.java
@@ -9,20 +9,20 @@ class GetterLazy {
     super();
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) ValueType getFieldName() {
-    java.lang.Object value = this.fieldName.get();
-    if ((value == null))
+    java.lang.Object $value = this.fieldName.get();
+    if (($value == null))
         {
           synchronized (this.fieldName)
             {
-              value = this.fieldName.get();
-              if ((value == null))
+              $value = this.fieldName.get();
+              if (($value == null))
                   {
                     final ValueType actualValue = new ValueType();
-                    value = ((actualValue == null) ? this.fieldName : actualValue);
-                    this.fieldName.set(value);
+                    $value = ((actualValue == null) ? this.fieldName : actualValue);
+                    this.fieldName.set($value);
                   }
             }
         }
-    return (ValueType) ((value == this.fieldName) ? null : value);
+    return (ValueType) (($value == this.fieldName) ? null : $value);
   }
 }

--- a/test/transform/resource/after-ecj/GetterLazyArguments.java
+++ b/test/transform/resource/after-ecj/GetterLazyArguments.java
@@ -18,107 +18,107 @@ class GetterLazyArguments {
     return null;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getField1() {
-    java.lang.Object value = this.field1.get();
-    if ((value == null))
+    java.lang.Object $value = this.field1.get();
+    if (($value == null))
         {
           synchronized (this.field1)
             {
-              value = this.field1.get();
-              if ((value == null))
+              $value = this.field1.get();
+              if (($value == null))
                   {
                     final String actualValue = stringInt("a", 1);
-                    value = ((actualValue == null) ? this.field1 : actualValue);
-                    this.field1.set(value);
+                    $value = ((actualValue == null) ? this.field1 : actualValue);
+                    this.field1.set($value);
                   }
             }
         }
-    return (String) ((value == this.field1) ? null : value);
+    return (String) (($value == this.field1) ? null : $value);
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getField2() {
-    java.lang.Object value = this.field2.get();
-    if ((value == null))
+    java.lang.Object $value = this.field2.get();
+    if (($value == null))
         {
           synchronized (this.field2)
             {
-              value = this.field2.get();
-              if ((value == null))
+              $value = this.field2.get();
+              if (($value == null))
                   {
                     final String actualValue = stringInt((true ? "a" : "b"), (true ? 1 : 0));
-                    value = ((actualValue == null) ? this.field2 : actualValue);
-                    this.field2.set(value);
+                    $value = ((actualValue == null) ? this.field2 : actualValue);
+                    this.field2.set($value);
                   }
             }
         }
-    return (String) ((value == this.field2) ? null : value);
+    return (String) (($value == this.field2) ? null : $value);
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getField3() {
-    java.lang.Object value = this.field3.get();
-    if ((value == null))
+    java.lang.Object $value = this.field3.get();
+    if (($value == null))
         {
           synchronized (this.field3)
             {
-              value = this.field3.get();
-              if ((value == null))
+              $value = this.field3.get();
+              if (($value == null))
                   {
                     final String actualValue = stringInt("a", (true ? 1 : 0));
-                    value = ((actualValue == null) ? this.field3 : actualValue);
-                    this.field3.set(value);
+                    $value = ((actualValue == null) ? this.field3 : actualValue);
+                    this.field3.set($value);
                   }
             }
         }
-    return (String) ((value == this.field3) ? null : value);
+    return (String) (($value == this.field3) ? null : $value);
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getField4() {
-    java.lang.Object value = this.field4.get();
-    if ((value == null))
+    java.lang.Object $value = this.field4.get();
+    if (($value == null))
         {
           synchronized (this.field4)
             {
-              value = this.field4.get();
-              if ((value == null))
+              $value = this.field4.get();
+              if (($value == null))
                   {
                     final String actualValue = stringRunnable(fun(), () -> {
 });
-                    value = ((actualValue == null) ? this.field4 : actualValue);
-                    this.field4.set(value);
+                    $value = ((actualValue == null) ? this.field4 : actualValue);
+                    this.field4.set($value);
                   }
             }
         }
-    return (String) ((value == this.field4) ? null : value);
+    return (String) (($value == this.field4) ? null : $value);
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getField5() {
-    java.lang.Object value = this.field5.get();
-    if ((value == null))
+    java.lang.Object $value = this.field5.get();
+    if (($value == null))
         {
           synchronized (this.field5)
             {
-              value = this.field5.get();
-              if ((value == null))
+              $value = this.field5.get();
+              if (($value == null))
                   {
                     final String actualValue = stringRunnable("a", () -> {
 });
-                    value = ((actualValue == null) ? this.field5 : actualValue);
-                    this.field5.set(value);
+                    $value = ((actualValue == null) ? this.field5 : actualValue);
+                    this.field5.set($value);
                   }
             }
         }
-    return (String) ((value == this.field5) ? null : value);
+    return (String) (($value == this.field5) ? null : $value);
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getField6() {
-    java.lang.Object value = this.field6.get();
-    if ((value == null))
+    java.lang.Object $value = this.field6.get();
+    if (($value == null))
         {
           synchronized (this.field6)
             {
-              value = this.field6.get();
-              if ((value == null))
+              $value = this.field6.get();
+              if (($value == null))
                   {
                     final String actualValue = (true ? stringInt((true ? "a" : "b"), (true ? 1 : 0)) : "");
-                    value = ((actualValue == null) ? this.field6 : actualValue);
-                    this.field6.set(value);
+                    $value = ((actualValue == null) ? this.field6 : actualValue);
+                    this.field6.set($value);
                   }
             }
         }
-    return (String) ((value == this.field6) ? null : value);
+    return (String) (($value == this.field6) ? null : $value);
   }
 }

--- a/test/transform/resource/after-ecj/GetterLazyBoolean.java
+++ b/test/transform/resource/after-ecj/GetterLazyBoolean.java
@@ -8,38 +8,38 @@
     return true;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) boolean isBooleanValue() {
-    java.lang.Object value = this.booleanValue.get();
-    if ((value == null))
+    java.lang.Object $value = this.booleanValue.get();
+    if (($value == null))
         {
           synchronized (this.booleanValue)
             {
-              value = this.booleanValue.get();
-              if ((value == null))
+              $value = this.booleanValue.get();
+              if (($value == null))
                   {
                     final boolean actualValue = calculateBoolean();
-                    value = actualValue;
-                    this.booleanValue.set(value);
+                    $value = actualValue;
+                    this.booleanValue.set($value);
                   }
             }
         }
-    return (java.lang.Boolean) value;
+    return (java.lang.Boolean) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) boolean isOtherBooleanValue() {
-    java.lang.Object value = this.otherBooleanValue.get();
-    if ((value == null))
+    java.lang.Object $value = this.otherBooleanValue.get();
+    if (($value == null))
         {
           synchronized (this.otherBooleanValue)
             {
-              value = this.otherBooleanValue.get();
-              if ((value == null))
+              $value = this.otherBooleanValue.get();
+              if (($value == null))
                   {
                     final boolean actualValue = (! calculateBoolean());
-                    value = actualValue;
-                    this.otherBooleanValue.set(value);
+                    $value = actualValue;
+                    this.otherBooleanValue.set($value);
                   }
             }
         }
-    return (java.lang.Boolean) value;
+    return (java.lang.Boolean) $value;
   }
   public @java.lang.Override @java.lang.SuppressWarnings("all") boolean equals(final java.lang.Object o) {
     if ((o == this))

--- a/test/transform/resource/after-ecj/GetterLazyEahcToString.java
+++ b/test/transform/resource/after-ecj/GetterLazyEahcToString.java
@@ -5,21 +5,21 @@
     super();
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) String getValue() {
-    java.lang.Object value = this.value.get();
-    if ((value == null))
+    java.lang.Object $value = this.value.get();
+    if (($value == null))
         {
           synchronized (this.value)
             {
-              value = this.value.get();
-              if ((value == null))
+              $value = this.value.get();
+              if (($value == null))
                   {
                     final String actualValue = "";
-                    value = ((actualValue == null) ? this.value : actualValue);
-                    this.value.set(value);
+                    $value = ((actualValue == null) ? this.value : actualValue);
+                    this.value.set($value);
                   }
             }
         }
-    return (String) ((value == this.value) ? null : value);
+    return (String) (($value == this.value) ? null : $value);
   }
   public @java.lang.SuppressWarnings("all") String getValue2() {
     return this.value2;

--- a/test/transform/resource/after-ecj/GetterLazyGenerics.java
+++ b/test/transform/resource/after-ecj/GetterLazyGenerics.java
@@ -8,37 +8,37 @@ class GetterLazyGenerics<E> {
     return null;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) E getField() {
-    java.lang.Object value = this.field.get();
-    if ((value == null))
+    java.lang.Object $value = this.field.get();
+    if (($value == null))
         {
           synchronized (this.field)
             {
-              value = this.field.get();
-              if ((value == null))
+              $value = this.field.get();
+              if (($value == null))
                   {
                     final E actualValue = getAny();
-                    value = ((actualValue == null) ? this.field : actualValue);
-                    this.field.set(value);
+                    $value = ((actualValue == null) ? this.field : actualValue);
+                    this.field.set($value);
                   }
             }
         }
-    return (E) ((value == this.field) ? null : value);
+    return (E) (($value == this.field) ? null : $value);
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) long getField2() {
-    java.lang.Object value = this.field2.get();
-    if ((value == null))
+    java.lang.Object $value = this.field2.get();
+    if (($value == null))
         {
           synchronized (this.field2)
             {
-              value = this.field2.get();
-              if ((value == null))
+              $value = this.field2.get();
+              if (($value == null))
                   {
                     final long actualValue = System.currentTimeMillis();
-                    value = actualValue;
-                    this.field2.set(value);
+                    $value = actualValue;
+                    this.field2.set($value);
                   }
             }
         }
-    return (java.lang.Long) value;
+    return (java.lang.Long) $value;
   }
 }

--- a/test/transform/resource/after-ecj/GetterLazyInAnonymousClass.java
+++ b/test/transform/resource/after-ecj/GetterLazyInAnonymousClass.java
@@ -7,21 +7,21 @@ public class GetterLazyInAnonymousClass {
         super();
       }
       public @java.lang.SuppressWarnings({"all", "unchecked"}) String getString() {
-        java.lang.Object value = this.string.get();
-        if ((value == null))
+        java.lang.Object $value = this.string.get();
+        if (($value == null))
             {
               synchronized (this.string)
                 {
-                  value = this.string.get();
-                  if ((value == null))
+                  $value = this.string.get();
+                  if (($value == null))
                       {
                         final String actualValue = "test";
-                        value = ((actualValue == null) ? this.string : actualValue);
-                        this.string.set(value);
+                        $value = ((actualValue == null) ? this.string : actualValue);
+                        this.string.set($value);
                       }
                 }
             }
-        return (String) ((value == this.string) ? null : value);
+        return (String) (($value == this.string) ? null : $value);
       }
     }
     x() {

--- a/test/transform/resource/after-ecj/GetterLazyNative.java
+++ b/test/transform/resource/after-ecj/GetterLazyNative.java
@@ -12,156 +12,156 @@ class GetterLazyNative {
     super();
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) boolean isBooleanField() {
-    java.lang.Object value = this.booleanField.get();
-    if ((value == null))
+    java.lang.Object $value = this.booleanField.get();
+    if (($value == null))
         {
           synchronized (this.booleanField)
             {
-              value = this.booleanField.get();
-              if ((value == null))
+              $value = this.booleanField.get();
+              if (($value == null))
                   {
                     final boolean actualValue = true;
-                    value = actualValue;
-                    this.booleanField.set(value);
+                    $value = actualValue;
+                    this.booleanField.set($value);
                   }
             }
         }
-    return (java.lang.Boolean) value;
+    return (java.lang.Boolean) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) byte getByteField() {
-    java.lang.Object value = this.byteField.get();
-    if ((value == null))
+    java.lang.Object $value = this.byteField.get();
+    if (($value == null))
         {
           synchronized (this.byteField)
             {
-              value = this.byteField.get();
-              if ((value == null))
+              $value = this.byteField.get();
+              if (($value == null))
                   {
                     final byte actualValue = 1;
-                    value = actualValue;
-                    this.byteField.set(value);
+                    $value = actualValue;
+                    this.byteField.set($value);
                   }
             }
         }
-    return (java.lang.Byte) value;
+    return (java.lang.Byte) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) short getShortField() {
-    java.lang.Object value = this.shortField.get();
-    if ((value == null))
+    java.lang.Object $value = this.shortField.get();
+    if (($value == null))
         {
           synchronized (this.shortField)
             {
-              value = this.shortField.get();
-              if ((value == null))
+              $value = this.shortField.get();
+              if (($value == null))
                   {
                     final short actualValue = 1;
-                    value = actualValue;
-                    this.shortField.set(value);
+                    $value = actualValue;
+                    this.shortField.set($value);
                   }
             }
         }
-    return (java.lang.Short) value;
+    return (java.lang.Short) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) int getIntField() {
-    java.lang.Object value = this.intField.get();
-    if ((value == null))
+    java.lang.Object $value = this.intField.get();
+    if (($value == null))
         {
           synchronized (this.intField)
             {
-              value = this.intField.get();
-              if ((value == null))
+              $value = this.intField.get();
+              if (($value == null))
                   {
                     final int actualValue = 1;
-                    value = actualValue;
-                    this.intField.set(value);
+                    $value = actualValue;
+                    this.intField.set($value);
                   }
             }
         }
-    return (java.lang.Integer) value;
+    return (java.lang.Integer) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) long getLongField() {
-    java.lang.Object value = this.longField.get();
-    if ((value == null))
+    java.lang.Object $value = this.longField.get();
+    if (($value == null))
         {
           synchronized (this.longField)
             {
-              value = this.longField.get();
-              if ((value == null))
+              $value = this.longField.get();
+              if (($value == null))
                   {
                     final long actualValue = 1;
-                    value = actualValue;
-                    this.longField.set(value);
+                    $value = actualValue;
+                    this.longField.set($value);
                   }
             }
         }
-    return (java.lang.Long) value;
+    return (java.lang.Long) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) float getFloatField() {
-    java.lang.Object value = this.floatField.get();
-    if ((value == null))
+    java.lang.Object $value = this.floatField.get();
+    if (($value == null))
         {
           synchronized (this.floatField)
             {
-              value = this.floatField.get();
-              if ((value == null))
+              $value = this.floatField.get();
+              if (($value == null))
                   {
                     final float actualValue = 1.0f;
-                    value = actualValue;
-                    this.floatField.set(value);
+                    $value = actualValue;
+                    this.floatField.set($value);
                   }
             }
         }
-    return (java.lang.Float) value;
+    return (java.lang.Float) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) double getDoubleField() {
-    java.lang.Object value = this.doubleField.get();
-    if ((value == null))
+    java.lang.Object $value = this.doubleField.get();
+    if (($value == null))
         {
           synchronized (this.doubleField)
             {
-              value = this.doubleField.get();
-              if ((value == null))
+              $value = this.doubleField.get();
+              if (($value == null))
                   {
                     final double actualValue = 1.0;
-                    value = actualValue;
-                    this.doubleField.set(value);
+                    $value = actualValue;
+                    this.doubleField.set($value);
                   }
             }
         }
-    return (java.lang.Double) value;
+    return (java.lang.Double) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) char getCharField() {
-    java.lang.Object value = this.charField.get();
-    if ((value == null))
+    java.lang.Object $value = this.charField.get();
+    if (($value == null))
         {
           synchronized (this.charField)
             {
-              value = this.charField.get();
-              if ((value == null))
+              $value = this.charField.get();
+              if (($value == null))
                   {
                     final char actualValue = '1';
-                    value = actualValue;
-                    this.charField.set(value);
+                    $value = actualValue;
+                    this.charField.set($value);
                   }
             }
         }
-    return (java.lang.Character) value;
+    return (java.lang.Character) $value;
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) int[] getIntArrayField() {
-    java.lang.Object value = this.intArrayField.get();
-    if ((value == null))
+    java.lang.Object $value = this.intArrayField.get();
+    if (($value == null))
         {
           synchronized (this.intArrayField)
             {
-              value = this.intArrayField.get();
-              if ((value == null))
+              $value = this.intArrayField.get();
+              if (($value == null))
                   {
                     final int[] actualValue = new int[]{1};
-                    value = ((actualValue == null) ? this.intArrayField : actualValue);
-                    this.intArrayField.set(value);
+                    $value = ((actualValue == null) ? this.intArrayField : actualValue);
+                    this.intArrayField.set($value);
                   }
             }
         }
-    return (int[]) ((value == this.intArrayField) ? null : value);
+    return (int[]) (($value == this.intArrayField) ? null : $value);
   }
 }

--- a/test/transform/resource/after-ecj/GetterLazyTransient.java
+++ b/test/transform/resource/after-ecj/GetterLazyTransient.java
@@ -6,21 +6,21 @@ class GetterLazyTransient {
     super();
   }
   public @java.lang.SuppressWarnings({"all", "unchecked"}) int getNonTransientField() {
-    java.lang.Object value = this.nonTransientField.get();
-    if ((value == null))
+    java.lang.Object $value = this.nonTransientField.get();
+    if (($value == null))
         {
           synchronized (this.nonTransientField)
             {
-              value = this.nonTransientField.get();
-              if ((value == null))
+              $value = this.nonTransientField.get();
+              if (($value == null))
                   {
                     final int actualValue = 1;
-                    value = actualValue;
-                    this.nonTransientField.set(value);
+                    $value = actualValue;
+                    this.nonTransientField.set($value);
                   }
             }
         }
-    return (java.lang.Integer) value;
+    return (java.lang.Integer) $value;
   }
   public @java.lang.SuppressWarnings("all") int getNonLazyTransientField() {
     return this.nonLazyTransientField;

--- a/test/transform/resource/after-ecj/SkipSuppressWarnings.java
+++ b/test/transform/resource/after-ecj/SkipSuppressWarnings.java
@@ -8,20 +8,20 @@ class SkipSuppressWarnings {
     return this.field;
   }
   public @java.lang.SuppressWarnings({"unchecked"}) String getField2() {
-    java.lang.Object value = this.field2.get();
-    if ((value == null))
+    java.lang.Object $value = this.field2.get();
+    if (($value == null))
         {
           synchronized (this.field2)
             {
-              value = this.field2.get();
-              if ((value == null))
+              $value = this.field2.get();
+              if (($value == null))
                   {
                     final String actualValue = "";
-                    value = ((actualValue == null) ? this.field2 : actualValue);
-                    this.field2.set(value);
+                    $value = ((actualValue == null) ? this.field2 : actualValue);
+                    this.field2.set($value);
                   }
             }
         }
-    return (String) ((value == this.field2) ? null : value);
+    return (String) (($value == this.field2) ? null : $value);
   }
 }

